pkg/controller/network_policy.go

```diff
@@ -196,6 +196,12 @@ func (c *Controller) handleUpdateNp(key string) error {
 		return err
 	}
 
+	namedPortMap, err := c.fetchNamedPorts(np.Namespace)
```

Comment:
We should move it to pod handler to reduce duplicated computation

Reply:
I refactored it